src/main/sensors/gyro.c

```diff
@@ -1158,6 +1200,11 @@ bool gyroOverflowDetected(void)
     return gyroSensor1.overflowDetected;
 }
 
+bool gyroYawSpinDetected(void)
+{
+    return gyroSensor1.yawSpinDetected;
+}
```

Comment:
Wrap in `#ifdef USE_`

Reply:
It compiles as is, and we know the gyro code works without them, so I  just copied what works.  :-)  usual dumb coder way of doing things.  Maybe @DieHertz knows the optimal way?  It's much beyond me.  To be honest all this code seemed excessively complex, but I wouldn't know.